### PR TITLE
update docs for new Docker for Mac

### PR DESCRIPTION
**Pull request by clouserw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,44 +51,23 @@ See some of our [Tips and Tricks](docs/README-DOCKER.md).
 
 #### First Thing's First
```
Comment (jaredhirsch) [inline README.md]: @clouserw While we're in here, could this be corrected to `First Things First`?

**Comment by jaredhirsch** [comment]
I'll take a look. Docker on Mac is really exciting

**Comment by lmorchard** [comment]
Might also want to toss a reference in there to [the docs to transition from Docker Toolkit to Docker for Mac](https://docs.docker.com/docker-for-mac/docker-toolbox/), if only at the end for us folks who already have the old-style setup running

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md]: will fix

**Comment by ghost** [comment]
New patch w/ comments addressed

**Comment by jaredhirsch** [comment]
Finally got Docker Toolbox uninstalled, new instructions work for me :tada:

**Comment by jaredhirsch** [comment]
Merging. Followup question: should we get rid of `docs/README-DOCKER.md`?

**Comment by lmorchard** [comment]
> Followup question: should we get rid of docs/README-DOCKER.md

I don't think so... though it could be better written or folded in elsewhere. It has a lot of useful info, all still relevant to Docker for Mac.

**Pull request merged**